src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesReducer = createSlice({
+  name: 'categories',
+  initialState: [],
+  reducers: {
+    checkStatus() {
+      return 'Under construction';
+    },
+  },
+});
+
+export const { checkStatus } = categoriesReducer.actions;
+export default categoriesReducer;
```

Comment:
- Kindly make sure to export the reducer, like `export default categoriesReducer.reducer;`

Reply:
Hello @GedeonTS 

Thank you for your kind feedback. I shall make the changes and ask for a review again.

Regards